pkg/rpcclient/policy/policy.go

```diff
@@ -0,0 +1,219 @@
+/*
+Package policy allows to work with the native PolicyContract contract via RPC.
+
+Safe methods are encapsulated into ContractReader structure while Contract provides
+various methods to perform PolicyContract state-changing calls.
+*/
+package policy
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/core/native/nativenames"
+	"github.com/nspcc-dev/neo-go/pkg/core/state"
+	"github.com/nspcc-dev/neo-go/pkg/core/transaction"
+	"github.com/nspcc-dev/neo-go/pkg/neorpc/result"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient/unwrap"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+// Invoker is used by ContractReader to call various methods.
+type Invoker interface {
+	Call(contract util.Uint160, operation string, params ...interface{}) (*result.Invoke, error)
+}
+
+// Actor is used by Contract to create and send transactions.
+type Actor interface {
+	Invoker
+
+	MakeCall(contract util.Uint160, method string, params ...interface{}) (*transaction.Transaction, error)
+	MakeRun(script []byte) (*transaction.Transaction, error)
+	MakeUnsignedCall(contract util.Uint160, method string, attrs []transaction.Attribute, params ...interface{}) (*transaction.Transaction, error)
+	MakeUnsignedRun(script []byte, attrs []transaction.Attribute) (*transaction.Transaction, error)
+	SendCall(contract util.Uint160, method string, params ...interface{}) (util.Uint256, uint32, error)
+	SendRun(script []byte) (util.Uint256, uint32, error)
+}
+
+// Hash stores the hash of the native PolicyContract contract.
+var Hash = state.CreateNativeContractHash(nativenames.Policy)
+
+const (
+	execFeeSetter      = "setExecFeeFactor"
+	feePerByteSetter   = "setFeePerByte"
+	storagePriceSetter = "setStoragePrice"
+)
```

Comment:
Or maybe it's even better to declare method names inside the native contract itself and reuse them in the client?

Reply:
Client never imports `core/native`, so either it'd be some separate package (like `nativenames`), or we have what we have.

Reply:
Clients test cover these cases, so let it be as is.